### PR TITLE
Eve CI backport

### PR DESCRIPTION
**Pull request by ssalaues**

### CODE REVIEW

File: eve/workers/unit_and_feature_tests/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM spotify/kafka
```
Comment (LaurenSpiegel) [inline eve/workers/unit_and_feature_tests/Dockerfile]: why spotify kafka instead of the one we're now using in zenko?

File: eve/main.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          command: npm run --silent lint_md
+      - ShellCommand:
+          name: run static analysis tools on code
+          command: npm run --silent lint
+      - ShellCommand:
+          name: run unit tests
+          command: npm test
+      - ShellCommand:
+          name: run feature tests
+          command: npm run ft_test
```
Comment (LaurenSpiegel) [inline eve/main.yml]: what about the 'behavior' tests that are in the package.json ("bh_test": "mocha --recursive tests/behavior",)

**Review by LaurenSpiegel — DISMISSED** [review]

File: eve/workers/unit_and_feature_tests/Dockerfile (hunk 1 above)
Comment (ssalaues) [reply eve/workers/unit_and_feature_tests/Dockerfile]: Good point, when this was originally made, we didn't have those but I'll test it out with that.

Comment (rahulreddy) [reply eve/workers/unit_and_feature_tests/Dockerfile]: I remember having this conversation with Rayenne at that time. His goal was to port whatever we were doing in circle.yml at that time and then do the switch to our kafka later. I think setting it up with our kafka would require pulling in Zookeeper alpine, getting the quorum work, setting up topics etc. I recommend doing that as a different PR.

File: eve/main.yml (hunk 2 above)
Comment (ssalaues) [reply eve/main.yml]: We were matching the circle enviroment so I'm not sure why those aren't included. @rahulreddy Should we add in the behavior tests into the CI?

**Comment by rahulreddy** [comment]
@ssalaues Can  you do git commit -x so that it references your original commit.

File: eve/main.yml (hunk 2 above)
Comment (rahulreddy) [reply eve/main.yml]: Yes we should.

File: eve/workers/unit_and_feature_tests/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "aws-sdk": "2.147.0",
+    "backo": "^1.1.0",
+    "bucketclient": "scality/bucketclient",
+    "commander": "^2.11.0",
+    "eslint": "^2.4.0",
+    "eslint-config-airbnb": "^6.0.0",
+    "eslint-config-scality": "scality/Guidelines",
+    "eslint-plugin-react": "^4.2.3",
+    "joi": "^10.6",
+    "kafka-node": "^1.6.0",
```
Comment (rahulreddy) [inline eve/workers/unit_and_feature_tests/package.json]: Can you match the version of kafka-node

**Review by rahulreddy — DISMISSED** [review]

File: eve/workers/unit_and_feature_tests/package.json (hunk 3 above)
Comment (ssalaues) [reply eve/workers/unit_and_feature_tests/package.json]: I swear I changed that but I guess I lost it when I squashed. I'll fix it.

**Comment by jonathan-gramain** [comment]
Per @tcarmet DM, backbeat 7 CI is OK so we should be able to close this PR.

**Pull request closed without merging**